xml/ha_storage_basics.xml

```diff
@@ -0,0 +1,134 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<!-- Converted by suse-upgrade version 1.1 -->
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="cha.ha.storage.dlm">
+ <title>Distributed Lock Manager (DLM)</title>
+ <info>
+  <abstract>
+   <para>
+    The Distributed Lock Manager (DLM) coordinates disk access for clustered
+    file systems. It administers file locking to increase performance and
+    availability. The DLM in the kernel is the base component used by &ocfs;,
+    GFS2, cluster MD, and cluster LVM (lvmlockd) to provide active-active
+    storage at each respective layer. The Distributed Lock Manager (DLM)
+    coordinates disk access for clustered file systems. It administers file
+    locking to increase performance and availability. The DLM in the kernel is
```

Comment:
The sentence "The DLM in the kernel..." occurs twice (see some lines above).

Reply:
Thanks! Actually, all sentences in the abstract did appear twice. ;) Fixed now.

Reply:
I incline to delete "The Distributed Lock Manager (DLM) coordinates disk access for clustered file systems. It administers file locking to increase performance and availability." since DLM is not used by file system solely. It is use by Cluster MD and Cluster LVM at the block level ;)